recipe/meta.yaml

```diff
@@ -32,7 +32,7 @@ about:
   home: https://github.com/eukaryote/pytest-tornasync
   license: MIT
   license_family: MIT
-  license_file: {{ RECIPE_DIR }}/LICENSE
+  license_file: LICENSE
```

Comment:
Next time please bump the build number

Reply:
Are you sure this would be a good idea? This is just cleaning up the recipe a bit without changing the final package at all (well, except for the meta-data). Seems wasteful to generate a new package just because of this.

Reply:
Just a reminder in general. In this case it will be ok. But the process here will be the same, the only difference that the package will be replaced by this new one. 
As both package have the license specified that is ok, if one packed had the license and the another one hadn't, that would be better to bump the recipe.
To avoid to "think" about those things, I recommend to always bump the build number, just in case ;)
But that is fine
:)

Reply:
Fair enough. 👍